README.md

```diff
@@ -8,7 +8,52 @@ The Python [OpenTelemetry](https://opentelemetry.io/) client.
 
 ## Installation
 
-## Usage
+This repo include multiple installable packages. The `opentelemetry-api`
+package includes abstract classes that comprise the OpenTelemetry API following
+[the
+specification](https://github.com/open-telemetry/opentelemetry-specification).
+The `opentelemetry-sdk` package is the reference implementation of the API.
+
+To install the API and SDK packages, fork or clone this repo and do an
+[editable
```

Comment:
Why do we recommend the editable install?

Reply:
Only because we haven't pushed anything to PyPI yet. I don't have a strong opinion on this vs. `python setup.py install`, but it looks like there are [a few good reasons](https://stackoverflow.com/questions/30306099/pip-install-editable-vs-python-setup-py-develop) to prefer `pip` here.